site/gatsby-site/src/pages/apps/submitted.js

```diff
@@ -79,78 +90,101 @@ const SubmittedIncidentsPage = ({ ...props }) => {
 
   return (
     <>
-      <AiidHelmet path={props.location.pathname}>
-        <title>{t('Submitted Incident Report List')}</title>
-      </AiidHelmet>
-      <div className={'titleWrapper'}>
-        <h1>
-          <Trans ns="submitted">Submitted Incident Report List</Trans>
-        </h1>
-      </div>
-      <div>
-        <SubmissionList />
-        <div className="mt-5">
-          <h2>
-            <Trans ns="submitted">Quick Add URLs</Trans>
-          </h2>
-          <p>
-            <Trans ns="submitted" i18nKey="quickaddDescription">
-              These reports were added anonymously by users in the Quick Add form on the landing
-              page
-            </Trans>
-          </p>
-          <ListGroup className="mb-5">
-            {sortedQuickAdds.length < 1 && <ListSkeleton />}
-            {sortedQuickAdds.map(({ _id, url, date_submitted }) => (
-              <div
-                key={_id}
-                className="flex flex-row md:flex-wrap flex-nowrap border-b last:border-none m-0 p-2"
-              >
-                <div className="flex">
-                  <div className="flex items-center mr-10">
-                    <Button
-                      variant="outline-secondary"
-                      disabled={!isAdmin}
-                      onClick={() => {
-                        if (
-                          window.confirm(t('Are you sure you want to delete “{{url}}”?', { url }))
-                        ) {
-                          submitDeleteQuickAdd(_id);
-                        }
-                      }}
-                    >
-                      <Trans>Delete</Trans>
-                      <svg
-                        aria-hidden="true"
-                        className="ml-2 -mr-1 w-4 h-4"
-                        fill="currentColor"
-                        viewBox="0 0 20 20"
-                        xmlns="http://www.w3.org/2000/svg"
+      {pageLoading ? (
+        <ListSkeleton />
+      ) : (
+        <>
+          {id ? (
+            <>
+              <AiidHelmet path={props.location.pathname}>
```

Comment:
I think the <helmet> should be outside the `pageLoading` conditional. Otherwise, the initial HTML will render with no title.